lms/models/application_instance.py

```diff
@@ -87,15 +87,19 @@ class Product(str, Enum):
     #: (each token belongs to a different user of this application
     #: instance's LMS).
     access_tokens = sa.orm.relationship(
-        "OAuth2Token", back_populates="application_instance"
+        "OAuth2Token",
+        back_populates="application_instance",
+        foreign_keys="OAuth2Token.consumer_key",
```

Comment:
This `foreign_keys` is needed on the SQLA side to know what FK should the relationship follow in the backwards relationship to access_tokens and group_infos now that there are two FK pointing to `application_instances`